client-src/src/pages/inq-form1/inq-form1.ts

```diff
@@ -76,15 +95,16 @@ export class InqForm1Page {
   }
 
   logForm1() {
+    this.inqForm.value.address.city = this.city;
+    this.inqForm.value.address.state = this.state;
+    this.inqForm.value.address.country = this.country;
     if(this.inqForm.valid){
       console.log("Form to be logged", this.inqForm.value);
       this.presentLoadingCustom();
-      // setTimeout(()=>{this.navCtrl.push(InqForm2Page);},2000);
       this.inqProvider.createInq(this.inqForm.value)
         .subscribe(
         data => { 
           this.responseData = data;
-
           if(this.responseData.data){
             this.notify.showInfo("Inquiry Registered Successfully");
```

Comment:
Instead of hard-coding, create a service of constants and pick messages from there. This will help us in standard outputs and reusability.

Reply:
Example constant name: INQUIRY.REGISTRATION.SUCCESS

Reply:
Addressed in commit 085b3c9